.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: UI Test
```

Comment:
Can probably include this in .gitignore

Reply:
Actually this file is a GitHub Actions file that needs to be present in upstream. See the [documentation](https://docs.github.com/en/actions) for more info.